src/PrestaShopBundle/DependencyInjection/AddOnsConfiguration.php

```diff
@@ -35,6 +36,8 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $rootNode = $treeBuilder->root('prestashop');
 
+        Tools::refreshCACertFile();
```

Comment:
@Quetzacoalt91 It will be refreshed only when the cache will have to be warmed up.